.travis.yml

```diff
@@ -42,5 +67,5 @@ deploy:
   provider: pages
   skip_cleanup: true
   github_token: $GITHUB_TOKEN
-  on:
+  "on":
```

Comment:
I vaguely remember doing something like this in the past, but my quick searches couldn't come up with anything. What's the story behind this?

Reply:
it is from the yaml lint bear.
I used to disabled it in git-task-list. But it is far easier just to quote it.

Reply:
I guess you did it in gci-leader?

Reply:
yup @andrewda did. https://github.com/coala/gci-leaders/commit/dff8ba5094d165a821f60d6bcfcf3385c28d5bba